docs/docs/guides/sign-in-with-github-google-facebook-linkedin.mdx

```diff
@@ -1001,6 +1001,82 @@ selfservice:
 Next, open the login endpoint of the SecureApp and you should see the Yandex
 Login option!
 
+## Apple
+
+To set up "Sign in with Apple" you must
+[create an app, a service and a private key key](https://developer.okta.com/blog/2019/06/04/what-the-heck-is-sign-in-with-apple#create-an-app-id).
```

Comment:
```suggestion
[create an app, a service and a private key](https://developer.okta.com/blog/2019/06/04/what-the-heck-is-sign-in-with-apple#create-an-app-id).
```

Reply:
fixed in https://github.com/ory/kratos/pull/1782/commits/23bde7c7e9f07635ddf1f9e63019bced48662eee